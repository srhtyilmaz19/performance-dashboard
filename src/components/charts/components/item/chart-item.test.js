import React from "react";

import { cleanup, render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import ChartItem from "./chart-item";

afterEach(cleanup);

describe("<ChartItem />", () => {
  it("should render error component", function () {
    const { asFragment } = render(<ChartItem metrics={{}} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
