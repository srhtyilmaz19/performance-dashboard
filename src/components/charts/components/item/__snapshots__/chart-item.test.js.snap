// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChartItem /> should render error component 1`] = `
<DocumentFragment>
  <p>
    NOT VALID DATA
  </p>
</DocumentFragment>
`;
